docs/DISCOVERABLE_PARTITIONS.md

```diff
@@ -1 +0,0 @@
-[This content has moved to the UAPI group website](https://uapi-group.org/specifications/specs/discoverable_partitions_specification/)
```

Comment:
This page provided a redirect to maintain url stability. This must either be provided via a different mechanism or this page must remain.

Reply:
I thought this was already done https://github.com/systemd/systemd/pull/31648/commits/7d41cd2fa88eed3b701fbc1de9b30c5767c59dae
But I can easily revert the commit :)

Reply:
I don't understand. I see `docs/DISCOVERABLE_PARTITIONS.md` in the current `main`. https://github.com/systemd/systemd/commit/7d41cd2fa88eed3b701fbc1de9b30c5767c59dae was never merged?

Reply:
Sure, it is a part of this PR :) I reverted it and the page remains